frontend/src/metabase/databases/components/DatabaseSyncModal/DatabaseSyncModal.tsx

```diff
@@ -14,12 +18,39 @@ import {
   ModalCloseIcon,
 } from "./DatabaseSyncModal.styled";
 
+const getSampleUrl = (candidates: DatabaseCandidate[]) => {
+  const tables = candidates.flatMap(d => d.tables);
+  const table =
+    tables.find((t: TableCandidate) => t.title.includes("Orders")) ?? tables[0];
+  return table?.url;
+};
+
+const useSampleDatabaseLink = () => {
+  const { data: databases } = useDatabaseListQuery();
+  const xraysEnabled = useSetting("enable-xrays");
+  const sampleDatabase = databases?.find(d => d.is_sample);
+
+  const { data: databaseCandidates } = useListDatabaseCandidatesQuery(
+    { id: sampleDatabase?.id },
+    { skip: !sampleDatabase || !xraysEnabled },
+  );
+
+  const sampleUrl = databaseCandidates
+    ? getSampleUrl(databaseCandidates)
+    : undefined;
+
+  return sampleUrl;
+};
```

Comment:
this is logic that got ported from the deleted container/DatabaseSyncModal. I particularly love how it's now a nice little hook instead of a tangled web of fetching HoCs and selectors.